core/config/src/main/java/org/apache/causeway/core/config/CausewayConfiguration.java

```diff
@@ -957,194 +981,182 @@ public static class DomainEvent {
+                        "previous.*:fa-backward-step",
+                        "next.*:fa-forward-step",
 
-                            "approve.*:fa-regular fa-thumbs-up",
-                            "reject.*:fa-regular fa-thumbs-down",
+                        "approve.*:fa-regular fa-thumbs-up",
+                        "reject.*:fa-regular fa-thumbs-down",
 
-                            "verify.*:fa-solid fa-check",
-                            "decline.*:fa-solid fa-xmark",
-                            "cancel.*:fa-solid fa-xmark",
+                        "verify.*:fa-solid fa-check",
+                        "decline.*:fa-solid fa-xmark",
+                        "cancel.*:fa-solid fa-xmark",
 
-                            "discard.*:fa-regular fa-trash-can",
+                        "discard.*:fa-regular fa-trash-can",
 
-                            "assign.*:fa-regular fa-hand-point-right",
+                        "assign.*:fa-regular fa-hand-point-right",
 
-                            "calculate.*:fa-calculator",
+                        "calculate.*:fa-calculator",
 
-                            "import.*:fa-solid fa-file-import",
-                            "export.*:fa-solid fa-file-export",
+                        "import.*:fa-solid fa-file-import",
+                        "export.*:fa-solid fa-file-export",
 
-                            "first.*:fa-regular fa-star",
+                        "first.*:fa-regular fa-star",
 
-                            "install.*:fa-solid fa-wrench",
+                        "install.*:fa-solid fa-wrench",
 
-                            "setup.*:fa-solid fa-gear",
-                            "configure.*:fa-solid fa-gear",
+                        "setup.*:fa-solid fa-gear",
+                        "configure.*:fa-solid fa-gear",
 
-                            "refresh.*:fa-sync",
-                            "renew.*:fa-rotate-right",
-                            "reset.*:fa-rotate-left",
+                        "refresh.*:fa-sync",
+                        "renew.*:fa-rotate-right",
+                        "reset.*:fa-rotate-left",
 
-                            "save.*:fa-regular fa-floppy-disk",
+                        "save.*:fa-regular fa-floppy-disk",
 
-                            "switch.*:fa-exchange",
-                            "random.*:fa-shuffle",
+                        "switch.*:fa-exchange",
+                        "random.*:fa-shuffle",
 
-                            "view.*:fa-regular fa-eye",
+                        "view.*:fa-regular fa-eye",
 
-                            "wizard.*:fa-solid fa-wand-magic-sparkles"
+                        "wizard.*:fa-solid fa-wand-magic-sparkles"
 
-                    };
+                    })
+                    String[] patterns) {
 
-                    @Getter(lazy = true)
-                    private final Map<Pattern, String> patternsAsMap = asMap(getPatterns());
+                    //(lazy = true) //TODO
```

Comment:
Perhaps use a `_StableValue` here, but would need an additional record component, that is not to be exposed with the default constructor as seen by Spring (`[@ConstructorBinding](https://docs.spring.io/spring-boot/4.0.0-M1/api/java/org/springframework/boot/context/properties/bind/ConstructorBinding.html)).